src/main/java/frc/robot/subsystems/ball/feeder/UltrasonicPoller.java

```diff
@@ -9,22 +10,50 @@
 class UltrasonicPoller extends RepeatingPooledSubsystem {
 
     Ultrasonic ultrasonicSensor;
+    StopWatch timer = new StopWatch();
```

Comment:
Pulling in a whole Apache Commons dependency just for this is way, way unnecessary.

Reply:
How about using a `long` and `System.nanoTime` instead of this stopwatch? Test out `long last = System.nanoTime(); Thread.sleep(1000); long current = System.nanoTime(); long elapsed = current - last;` and see how this might be useful.

Reply:
I changed it to use System.nanoTime. Take a look at it, I'm pretty sure it should work.